apps/core/mix.exs

```diff
@@ -25,7 +25,7 @@ defmodule Core.Mixfile do
   def application do
     [
       mod: {Core.Application, []},
-      extra_applications: [:logger, :runtime_tools, :yamerl, :worldly]
+      extra_applications: [:logger, :runtime_tools]
     ]
```

Comment:
Please remove `uuid` and `excountries` from deps and restrict `yamerl` and `worldly` to `only: dev`